internal/httpclient/docs/OAuth2API.md

```
@@ -1369,6 +1370,7 @@ Name | Type | Description  | Notes
  **grantType** | **string** |  | 
  **clientId** | **string** |  | 
  **code** | **string** |  | 
+ **codeVerifier** | **string** |  | 
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)